pipelines/docker-build-multi-platform-oci-ta.yaml

```diff
@@ -150,6 +150,8 @@ spec:
       value: $(params.output-image).prefetch
     - name: ociArtifactExpiresAfter
       value: $(params.image-expires-after)
+    - name: dev-package-managers
+      value: "true"
```

Comment:
This change will enable this for all konflux apps that use pipelines/docker-build-multi-platform-oci-ta.yaml, if this is not what we want it may be better to define a new parameter in [params](https://github.com/securesign/pipelines/blob/dee9337eef0a0e681008d5cec1be3911d1d38fb4/pipelines/docker-build-multi-platform-oci-ta.yaml#L20)

and pass it in to this pipeline from what ever application you want to enable this for.